src/ws/context.rs

```diff
@@ -133,7 +133,7 @@ where
 {
     /// Write payload
     #[inline]
-    fn write(&mut self, data: Binary) {
+    pub fn write(&mut self, data: Binary) {
```

Comment:
Do you think it may be wise for us to introduce wrapper type for websocket frame, to avoid potential misuse?(e.g. user fill Binary himself)

Reply:
I guess if `ws::Frame::message()` would return a dedicated type and `WebsocketContext::write()` would only accept that type it might be useful. I'm not sure though if `ws::Frame::message()` is used anywhere else though.

Reply:
I think if we want to expose this API, it should be dedicated type to avoid potential misuse, as you imagine not all people would be careful enough when reading documentation :)

You could leave this `write` method for internal use and instead create new method `send_frame` or just `frame` that would accept wrapper type and pass it's Binary data into `write`

Reply:
I've introduced a `FramedBinary` type now for this purpose. This is technically a breaking change, but since master is using a newer `actix` version the next release would be a breaking release anyway 😉